### PR TITLE
Update code review checklist to reflect lessons learned

### PR DESCRIPTION
**Pull request by waldoj**
Now that we've reviewed a half-dozen sprints, Curtis and I have found that aspects of how we work and the how the vendor works don't align with the documented process, in ways that require modifying the checklist, rather than the process. These changes codify the reality of how we've found it's best for us to work.

### CODE REVIEW

File: code-review.md [hunk 1]
```diff
@@ -76,21 +76,17 @@ We use this list when performing a code review to ensure that all tasks have bee
...
 	- [ ] read through every task on the Scrum board that's encompassed by this pull request
 	- [ ] read the description of every commit that comprises the pull request
-- [ ] deploy the site on your system
+- [ ] deploy the site on your local system
 	- [ ] test all functionality in all major browsers, emphasizing the functionality that this pull request addresses
 		- [ ] the browser to test the most thoroughly is Internet Explorer 11 on Windows 10
-		- [ ] Chrome, Firefox, Safari, mobile Chrome, and mobile Safari
-	- [ ] use an automated audit tool for code quality and practices (recommended: Chrome Audits)
+		- [ ] also test in Chrome and Firefox
+	- [ ] use an automated audit tool for code quality and practices (recommended: Chrome Audit)
```
[the remark this answers is not in the document]
Comment (waldoj) [reply code-review.md]: There are, but Chrome's tool is by far the best. IE11 is nearly five years old now, so using its tools will land you with a state-of-the-art website for 2013. :)  But thinking through the various things that Chrome tests, I can't think of any that are Chrome-specific, in terms of the rendering engine or supported functionality. _Maybe_ some of the Progressive Web App tests, but those aren't applicable to our work here, anyway. So using Chrome's test tools shouldn't be specific to Chrome in any meaningful way — it's really just about best website practices.

[the remark this answers is not in the document]
Comment (sztaylorakgov) [reply code-review.md]: I get that.  I'm not thinking about testing w/ Chrome means something only works with Chrome.  I'm thinking more about trying to keep our dev toolkit streamlined to supported standards.  Understanding the tools supported by either IE 11 or Edge (on our roadmap at DHSS) is a legitimate and honest question aimed at reducing the overhead for maintaining the app.

[the remark this answers is not in the document]
Comment (waldoj) [reply code-review.md]: As a macOS user, I'm afraid that my experience is limited here. My use of Edge is limited to "yes, this new website feature is working as expected in Edge." I know that the developer tools in IE11 stopped being updated in any way a few years ago, with Microsoft putting their efforts into Edge. Microsoft provides some great [documentation of developer tools](https://developer.microsoft.com/en-us/microsoft-edge/tools/), with [a specific guide to using them](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide).

It's not clear to me that Edge's tools are as good as Chrome's tools, but Microsoft seems to be putting a lot of resources into Edge. Edge is basically no website developer's primary browser, and Microsoft wants to change that.

File: code-review.md [hunk 2]
```diff
@@ -76,21 +76,17 @@ We use this list when performing a code review to ensure that all tasks have bee
 	- [ ] read the description of the pull request, which should summarize the changes made
 	- [ ] read through every task on the Scrum board that's encompassed by this pull request
 	- [ ] read the description of every commit that comprises the pull request
-- [ ] deploy the site on your system
+- [ ] deploy the site on your local system
 	- [ ] test all functionality in all major browsers, emphasizing the functionality that this pull request addresses
 		- [ ] the browser to test the most thoroughly is Internet Explorer 11 on Windows 10
-		- [ ] Chrome, Firefox, Safari, mobile Chrome, and mobile Safari
-	- [ ] use an automated audit tool for code quality and practices (recommended: Chrome Audits)
+		- [ ] also test in Chrome and Firefox
```
[the remark this answers is not in the document]
Comment (waldoj) [reply code-review.md]: Oh, great—I didn't know that it was on the roadmap! We'll start testing accordingly.

**Pull request merged**